scripts/install_third_party_libs.py

```diff
@@ -89,6 +116,59 @@ def get_yarn_command():
     return 'yarn'
 
 
+def install_buf_and_protoc():
+    """Installs buf and protoc for Linux or Darwin, depending upon the
+    platform.
+    """
+    if os.path.exists(BUF_DIR):
+        return
+
+    buf_files = BUF_DARWIN_FILES if common.is_mac_os() else BUF_LINUX_FILES
+    protoc_file = (
+        PROTOC_DARWIN_FILE if common.is_mac_os() else PROTOC_LINUX_FILE)
+
+    common.ensure_directory_exists(BUF_DIR)
+    for bin_file in buf_files:
+        python_utils.url_retrieve('%s/%s' % (
+            BUF_BASE_URL, bin_file), filename=os.path.join(BUF_DIR, bin_file))
+    python_utils.url_retrieve('%s/%s' % (
+        PROTOC_URL, protoc_file), filename=os.path.join(BUF_DIR, protoc_file))
+    try:
+        with zipfile.ZipFile(os.path.join(BUF_DIR, protoc_file), 'r') as zfile:
+            zfile.extractall(path=PROTOC_DIR)
+        os.remove(os.path.join(BUF_DIR, protoc_file))
+    except Exception:
+        raise Exception('Error installing protoc binary')
```

Comment:
There seem to be some issues with this on GitHub Actions?

Reply:
It should be fixed now.